src/productCounter.js

```diff
@@ -0,0 +1,10 @@
+const productCounter = () => {
+  const prodTempl = document.querySelectorAll('.card-product');
+  const counterDis = document.querySelector('.item-counter');
+  counterDis.textContent = `(${prodTempl.length})`;
+
+  console.log(prodTempl.length);
```

Comment:
- [ ] good job on changing the suggestions the previous CR made 👍 however in the previous review was also mentioned to remove this `console.log`.